tokio/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.24.1 (January 6, 2022)
+
+This release fixes a compilation failure when using ([#5356])
+```
+RUSTFLAGS='--cfg tokio_no_atomic_u64 --cfg tokio_no_const_mutex_new'
+```
```

Comment:
I think it would be clearer to mention cases where both of these would actually be enabled. Something like:

```suggestion
This release fixes a compilation failure on targets without `AtomicU64` when using rustc older than 1.63. ([#5356])
```